op-guide/configuration.md

```diff
@@ -0,0 +1,52 @@
+# Configuration flags
+
+TiDB/TiKV/PD are configurable through command-line flags and environment variables.
+
+
+##TiDB
+
+The default TiDB ports are 4000 for client requests and 10080 for status report.
+
+### --store
++ Storage engine type
```

Comment:
Please note the fowllowing two changes because it's not a full sentence (without the predicate)：
1. No need to capitalize the first letter.
2. No full stop at the end unless there is one or more sentences behind it.